### PR TITLE
OSX daemons

### PR DESCRIPTION
**Pull request by ubozov**
Resolves BV-889.

### CODE REVIEW

File: unix/daemon_darwin.go [hunk 1]
```diff
@@ -0,0 +1,133 @@
...
+
+// Stop stops the daemon.
+func (d *Daemon) Stop() error {
+	cmd := exec.Command("launchctl", "unload", d.path())
+	return cmd.Run()
+}
+
+// Remove removes the daemon.
+func (d *Daemon) Remove() error {
+	exec.Command("launchctl", "remove", d.name()).Run()
```
Comment (sofabeat) [inline unix/daemon_darwin.go]: Can this command fail? Need error handling?

Comment (ubozov) [reply unix/daemon_darwin.go]: Thanks. This command was redundant, it was deleted

File: unix/daemon_darwin.go [hunk 2]
```diff
@@ -0,0 +1,130 @@
...
+	"os"
+	"os/exec"
+	"os/user"
+	"path/filepath"
+	"regexp"
+	"text/template"
+)
+
+// Daemon has a parameters for OSX daemon.
+type Daemon struct {
```
Comment (furkhat) [inline unix/daemon_darwin.go]: struct declaration is duplicated in both files. Does it make sense to have it in some non-platform based file? (for ex: `types.go`)

Comment (ubozov) [reply unix/daemon_darwin.go]: updated, placed to daemon.go

**Comment by ubozov** [comment]
@furkhat, go packages were updated

**Pull request merged**